src/styles/index.ts

```diff
@@ -1377,6 +1389,10 @@ const styles = (theme: ThemeColors) =>
             color: theme.textSupporting,
         },
 
+        lh14: {
+            lineHeight: 14,
```

Comment:
Where is this used exactly? I feel like we already have `lineHeightSmall` for this no?